pkg/template/static_context.go

```diff
@@ -420,3 +423,41 @@ func (ctx StaticCtx) isKurl() bool {
 
 	return configMap != nil
 }
+
+func (ctx StaticCtx) distribution() string {
```

Comment:
would it be better to have this return an array of strings? If something can be both "AWS" and "OpenShift" for instance

Reply:
per marc, Openshift is the proper return here